src/bokeh/util/warnings.py

```diff
@@ -23,13 +23,25 @@
 # Imports
 #-----------------------------------------------------------------------------
 
+# Standard library imports
+import inspect
+import os
+import warnings  # lgtm [py/import-and-import-from]
```

Comment:
Probably this `lgtm` comment isn't necessary anymore.
```suggestion
import warnings
```

Reply:
definitely, but I am going to remove all of them in a PR later this week, so there's no need to task @Hoxbro more